docs/reference/protocol/index.rst

```diff
@@ -18,6 +18,74 @@ clients and servers.  The protocol is supported over TCP/IP.
     dataformats
 
 
+.. _ref_protocol_connecting:
+
+Connecting to EdgeDB 
+====================
+
+The EdgeDB binary protocol has two modes of operation over TCP/IP: 
```

Comment:
```suggestion
The EdgeDB binary protocol has two modes of operation over TCP/IP: 
```

Technically, HTTP isn't always TCP (HTTP/3 is UDP, for example), so I'd reword this.

Reply:
No, the regular socket binary protocol is TCP.